recipe/deploy/writable.php

```diff
@@ -108,8 +108,8 @@
         } elseif (commandExist('setfacl')) {
             $setFaclUsers = "-m u:\"$httpUser\":rwX";
             // Check if remote user exists, before adding it to setfacl
-            $remoteUserId = run("id -u $remoteUser &>/dev/null 2>&1 || exit 0");
-            if (!empty($remoteUserId)) {
+            $remoteUserExists = test("id -u $remoteUser &>/dev/null 2>&1 || exit 0");
```

Comment:
Let's also delete `exit 0` and use 
```
run('...', ['no_throw' => true]);
```

Reply:
@antonmedv Does that mean I shall replace `test()` with `run()` again, as `test()` doesn't have any additional options? 

If I just remove `|| exit 0` and use it with `test()`, it keeps working. I still get `true` if the user exists and `false` if it doesn't.

Reply:
Ohh, I see, sorry for misleading. Let me think a bit. Maybe test() should also have options?

Reply:
@antonmedv 
I am not sure if that's necessary.  I tried all variants to see how it works:

If the user does exist
1. `run("id -u $remoteUser &>/dev/null 2>&1")`: Returns empty string -> the deployment continues
2. `run("id -u $remoteUser &>/dev/null 2>&1", ['no_throw' => true])`: Returns empty string -> the deployment continues
3. `test("id -u $remoteUser &>/dev/null 2>&1")`: Returns true -> the deployment continues

If the user doesn't exist
1. `run("id -u $remoteUser &>/dev/null 2>&1")`: Throws exception -> the deployment is abortet with exit code 1.
2. `run("id -u $remoteUser &>/dev/null 2>&1", ['no_throw' => true])`: Returns empty string -> the deployment continues
3. `test("id -u $remoteUser &>/dev/null 2>&1")`: Returns false -> the deployment continues

So i'd say test() works as expected, but I am not aware of side effects you might see.